lib/ui/login/login_state.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'login_state.freezed.dart';
+
+@freezed
+class LoginState with _$LoginState {
```

Comment:
Please move it the Integrate task next time, you can simply run the API with a hardcoded information for POC 🙏

Reply:
Got it. I'll do it from next time. 🫡